service.rb

```diff
@@ -1,24 +1,22 @@
-require 'socket'
 require 'cgi'
 require 'uri'
 require 'yaml/store'
+require 'rack/handler/puma'
+require 'rack'
 
 require_relative './http_request'
 
 class Service
```

Comment:
I know this `Service` class could be removed and replaced with just the `app`. Lmk if I should just do that?